From 55e4316efae87aead24e53e65a69871a008e8a72 Mon Sep 17 00:00:00 2001
From: Lukas Rusak <lorusak@gmail.com>
Date: Wed, 13 Apr 2016 11:09:49 -0700
Subject: [PATCH] kodi: add patch to disable using the tv menu language when
 using CEC kodi-999.15-disable-using-tv-menu-language-by-default.patch

---
 system/peripherals.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system/peripherals.xml b/system/peripherals.xml
index 1fbb4bc924b..eee51755e2d 100644
--- a/system/peripherals.xml
+++ b/system/peripherals.xml
@@ -18,7 +18,7 @@
     <setting key="cec_wake_screensaver" type="bool" value="1" label="36010" order="7" />
     <setting key="standby_pc_on_tv_standby" type="enum" value="13011" label="36029" order="8" lvalues="36028|13005|13011|13010|13009|36044|36045" />
     <setting key="standby_tv_on_pc_standby" type="bool" value="1" label="36026" order="9" />
-    <setting key="use_tv_menu_language" type="bool" value="1" label="36018" order="10" />
+    <setting key="use_tv_menu_language" type="bool" value="0" label="36018" order="10" />
     <setting key="pause_playback_on_deactivate" type="bool" value="1" label="36033" configurable="0" />
     <setting key="pause_or_stop_playback_on_deactivate" type="enum" value="231" label="36033" order="11" lvalues="231|36044|36045" />
     <setting key="connected_device" type="enum" label="36019" value="36037" lvalues="36037|36038" order="12" />
